Readme.md

```diff
@@ -1,5 +1,9 @@
 # Bytes utility
 
+[![npm version](https://badge.fury.io/js/bytes.svg)](http://badge.fury.io/js/bytes)
+[![Dependency Status](https://www.versioneye.com/nodejs/bytes/2.1.0/badge.svg)](https://www.versioneye.com/nodejs/bytes/2.1.0)
```

Comment:
We have no dependencies. I don't see the purpose of having the dependency badge?